.github/actions/upload-coverage/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Upload Coverage
+description: Upload coverage to codecov
+
```

Comment:
Do you understand why this report from Codecov shows (what I believe is) incorrect coverage?

https://github.com/mfem/mfem/pull/2041#issuecomment-786411453